website/content/api-docs/connect/intentions.mdx

```diff
@@ -43,16 +43,7 @@ The table below shows this endpoint's support for
 
 | Blocking Queries | Consistency Modes | Agent Caching | ACL Required                   |
 | ---------------- | ----------------- | ------------- | ------------------------------ |
-| `NO`             | `none`            | `none`        | `intentions:write`<sup>1</sup> |
-
-<p>
-  <sup>1</sup> Intention ACL rules are specified as part of a{' '}
-  <code>service</code> rule. See{' '}
-  <a href="/consul/docs/connect/intentions#intention-management-permissions">
-    Intention Management Permissions
-  </a>{' '}
-  for more details.
-</p>
+| `NO`             | `none`            | `none`        | `intentions:write` <p> Define intention rules in the `service` policy. Refer to [ACL requirements for intentions](/consul/docs/connect/intentions/intentions-usage#acl-requirements) for additional information.</p> |
```

Comment:
is there a reason why we need to add `<p>` here?

Reply:
It adds a little padding and looks nicer than a br tag, IMO.